nss-day-cohort-38/react-nutshell-the-uniform-skyhooks
# navbar skeleton and login/logout added

to test:
git fetch --all
git checkout cn-navbar
npm install
npm run
open http://localhost:3000/ in chrome 

check the login/logout functionality. only the Home and Login options should be displayed when there is no authenticated user. Once logged in, the navbar should populate with the News, Events, Tasks, and Messages links. Also the Login link should be replaced by a Logout link.
